packages/snaps-sdk/src/types/methods/get-preferences.ts

```diff
@@ -13,9 +13,23 @@ export type GetPreferencesParams = never;
  * @property locale - The user's selected locale.
  * @property currency - The user's selected currency.
  * @property hideBalances - Whether the user has chosen to hide balances.
+ * @property useSecurityAlerts - Whether to run transactions and signatures through blockaid.
```

Comment:
Is this relevant for Snaps? They can't hook into Blockaid.

Reply:
Have no idea, maybe Solana team has plans to use blockaid? cc: @Montoya

Reply:
Yup they are integrating blockaid's API into the snap

Reply:
I think we should make it more generic and not mention any providers, our Snaps may choose to use Blockaid, but other Snaps could use other providers for "security alerts"

Reply:
Updated the description.